src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -458,6 +458,11 @@
 
 "ExposureNotificationSetting_Activate_OSENSetting_Description" = "For risk calculation, COVID-19 exposure notifications must be activated in the system settings so that exposures can be logged. Please activate COVID-19 exposure notifications in your system settings.";
 
+/* Home Navigation */
```

Comment:
Please dont modify english language files any more:)

Reply:
Apologies, I thought I'd reverted it already. I'll revert it back to the dev branch version now.